### ISSUE TITLE
Visualizer is all black (no colors)

### ISSUE THREAD

**Issue by shanet** [dpayne/cli-visualizer#1]
First off, thank ypu for putting this project together. The screenshots look incredible and I'm excited to get it working as I've always wished that ncmpcpp's visualizer was more colorful.

I cloned this repo, compiled, and installed the program, but when running `vis` the visualizer is all black bars as shown here:
![2015-12-27-173525_6720x1080_scrot](https://cloud.githubusercontent.com/assets/1159147/12013003/8463246c-acc1-11e5-9834-275aac455577.png)

My config files is the default with the colors option set to `rainbow`. Note that the off-black background color is coming from my XTerm settings. If I remove the `.Xresources` file, the background correctly reverts back to white.

Interestingly enough, when I comment out the colors option in the config file, the bars don't seem to max out, but are still black. Here's another screenshot showing this:
![2015-12-27-173556_6720x1080_scrot](https://cloud.githubusercontent.com/assets/1159147/12013004/87d63724-acc1-11e5-8242-9bef7dfb675c.png)

For what it's worth, the ncmpcpp visualizer works as intended with the correct color.

System info:
* Arch
* i3 WM
* XTerm

Any ideas why this may be happening? It could very well be something specific to my terminal environment. Any debugging steps I can take to determine the problem?

**Comment by dpayne** [comment]
Do you have a colors file, it should be under "~/.vis/colors/rainbow"?

**Comment by dpayne** [comment]
Also are you running a 256 color terminal? The one that works best is "rxvt-unicode".

**Comment by shanet** [comment]
I installed `rxvt-unicode` and the colors work as intended there. Yes, the rainbow colors file exists and is the same as the one the `install.sh` copies.

Looking more into my XTerm setup, I ran this color test Perl script: http://www.robmeerman.co.uk/unix/256colours. My output matches what a 256 color enabled terminal should print. No clue why I don't have colors on the visualizer when run through XTerm.
![2015-12-27-181058_6720x1080_scrot](https://cloud.githubusercontent.com/assets/1159147/12013158/e109e148-acc5-11e5-8c1b-e9a99b3f4b7d.png)

Thanks for you help though, at least I now know it's a problem with XTerm.
